pom.xml

```diff
@@ -90,7 +90,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
-            <version> 1.7.30</version> 
+            <version> 1.7.30</version>
```

Comment:
Should remove whitespace before the version number too, while you're cleaning this up.

Reply:
Thank you Jonathan, fixed.